cogs/info.py

```diff
@@ -44,13 +44,10 @@ async def trivia (self, ctx, arg1 = "", arg2 = ""):
         # *** ANSWER ***
         elif (arg1.lower() == "answer"):
             if (self.bot.trivia.check(arg2) == 0):
-                await ctx.send("Correct")
-                # Handle what should happen when a correct
-                # answer is submitted
+                await ctx.send(f"Answer: " + arg2 + " is correct!")
             else:
-                await ctx.send("Not Correct")
-                # Handle what should happen when an incorrect
-                # answer is submitted
+                await ctx.send(f"Sorry, answer: " + arg2 + " is incorrect.")
```

Comment:
This is probably sufficient for the design we have in place however, if we may want to expand this to track the number of submissions and end the game if a predefined limit has been reached.